ec/src/models/bw6/g2.rs

```diff
@@ -71,23 +81,34 @@ impl<P: BW6Config> From<G2Affine<P>> for G2Prepared<P> {
                 ell_coeffs_1.push(r.add_in_place(&q));
             }
         }
+        // TODO: this is probably the slowest part
```

Comment:
I think we can leave this TODO in, it's potential for even more future optimizations